docs/aws/audit/ec2monitoring/rules/ebs_snapshots_not_public.mdx

```diff
@@ -18,6 +18,29 @@ Security
 
 NIST, PCIDSS, ARPA, MAS
 
+### Remediation
+
+How to fix EC2 snapshots publicly accessible?
+
+#### Using AWS Console
+
+1. Log in to the AWS Management Console.
+2. Navigate to the Amazon EC2 service by searching for "EC2" in the AWS services search bar and selecting "EC2" from the results.
+3. In the EC2 dashboard, click on "Snapshots" in the left sidebar.
+4. Locate the snapshot(s) that are marked as public or that you want to secure.
```

Comment:
Similar to above. For each of the remediation documentation, we should add a line to highlight how a user can find the affected assets in Cloudanix console.